recipes/openssl/recipe.sh

```diff
@@ -23,26 +22,33 @@ BUILD_openssl=$BUILD_PATH/openssl/$(get_directory $URL_openssl)
 # function called for preparing source code if needed
 # (you can apply patch etc here.)
 function prebuild_openssl() {
+  cd $BUILD_PATH/openssl/openssl-${VERSION_openssl}
+  # check marker
+  if [ -f .patched ]; then
+    return
+  fi
+
   echo "patching $BUILD_PATH/openssl/openssl-${VERSION_openssl}/config"
-  try sed -i 's/android-armeabi/android-arm/g' $BUILD_PATH/openssl/openssl-${VERSION_openssl}/config
-  
-  # ad random seed & patch of rand_unix.c: https://mta.openssl.org/pipermail/openssl-users/2018-September/008860.html 
-  try patch -p1 $BUILD_PATH/openssl/openssl-${VERSION_openssl}/crypto/rand/rand_unix.c $RECIPE_openssl/patches/rand_unix.patch
+  try $SED 's/armv\[7-9\]\*-\*-android/armeabi-v7a\*-\*-android|armv\[7-9\]\*-\*-android/g' $BUILD_PATH/openssl/openssl-${VERSION_openssl}/config
```

Comment:
Somehow the build fails on x86, I wonder if it's because of this line?

Reply:
Fixed in https://github.com/opengisch/OSGeo4A/commit/b739f43c5f1489c4b2117795748d546e0d30f8ee would be nice if you could have a look